apps/i3wm/i3wm.talon

```diff
@@ -2,82 +2,81 @@
 os: linux
 tag: user.i3wm
 -
-port <number_small>: user.system_command("i3-msg workspace number {number_small}")
-(port flip | flipper): user.system_command("i3-msg workspace back_and_forth")
-port right: user.system_command("i3-msg workspace next")
-port left: user.system_command("i3-msg workspace prev")
+port <number_small>: user.i3wm_switch_to_workspace(number_small)
+(port flip | flipper): user.i3wm_switch_to_workspace("back_and_forth")
+port right: user.i3wm_switch_to_workspace("next")
+port left: user.i3wm_switch_to_workspace("prev")
```

Comment:
I guess if we override `user.desktop` we should override `user.desktop_next()` and `user.desktop_last()` too.